nmrfx-analyst/src/main/java/org/nmrfx/analyst/peaks/Multiplets.java

```diff
@@ -240,32 +239,27 @@ public static Optional<Multiplet> transferPeaks(Multiplet multiplet, List<Peak>
     }
 
     public static void addPeaksToMultiplet(Multiplet multiplet, double... ppms) {
-        int iDim = 0;
         PeakDim peakDim = getMultipletRoot(multiplet);
```

Comment:
The only way peakDim will be null is if the method parameter multiplet is null. Threw an IllegalArgumentException here, confirm if this is okay? or better to just log a message here.